internal/querynodev2/handlers.go

```diff
@@ -249,6 +251,72 @@ func (node *QueryNode) querySegments(ctx context.Context, req *querypb.QueryRequ
 	}, nil
 }
 
+func (node *QueryNode) optimizeSearchParams(ctx context.Context, req *querypb.SearchRequest, channel string) *querypb.SearchRequest {
+	log := log.Ctx(ctx).With(zap.String("channel", channel))
```

Comment:
Move this after line 259, avoid unnecessary operation

Reply:
done